pkg/test/fakes/mcpserver/Dockerfile.test_mcpserver

```diff
@@ -0,0 +1,5 @@
+FROM istionightly/base_debug
```

Comment:
The docker file for policybackend currently coexists with the Mixer docker files:
https://github.com/istio/istio/tree/master/mixer/docker

While I don't think this is fundamentally correct, it pays to be consistent. We should move this one to the Galley's docker folder.

Reply:
Is it in galley or pkg/mcp folder? The latter looks more appropriate to me.

Reply:
Galley. It is not the most appropriate place, but it follows the convention.
When we split the repos, we can move it to a better location.